From: Yoann Dubreuil <yoann.dubreuil@gmail.com>
Date: Thu, 15 Feb 2018 22:48:33 +0100
Subject: Reduce QuickSync encoding bitrate

Would have been better to add flags, but it's too late for tonight
---
 nageru/quicksync_encoder_impl.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/nageru/quicksync_encoder_impl.h b/nageru/quicksync_encoder_impl.h
index 5e215e5..0a0997f 100644
--- a/nageru/quicksync_encoder_impl.h
+++ b/nageru/quicksync_encoder_impl.h
@@ -202,10 +202,10 @@ private:
 	std::deque<ReferenceFrame> reference_frames;
 
 	// Static quality settings.
-	static constexpr unsigned int frame_bitrate = 15000000 / 60;  // Doesn't really matter; only initial_qp does.
+	static constexpr unsigned int frame_bitrate = 10000000 / 60;  // Doesn't really matter; only initial_qp does.
 	static constexpr unsigned int num_ref_frames = 2;
-	static constexpr int initial_qp = 15;
-	static constexpr int minimal_qp = 0;
+	static constexpr int initial_qp = 11;
+	static constexpr int minimal_qp = 11;
 	static constexpr int intra_period = 30;
 	static constexpr int intra_idr_period = MAX_FPS;  // About a second; more at lower frame rates. Not ideal.
 
